intro-HPC/ch_web_portal.tex

```diff
@@ -0,0 +1,217 @@
+\chapter{Using the HPC-UGent web portal}
+\label{ch:web_portal}
+
+\href{https://openondemand.org/}{\strong{Open OnDemand}} (or \texttt{OoD} for short)
+is a web portal to access HPC resources.
+
+You can use it to connect to the HPC-UGent infrastructure via:
+
+\begin{center}\url{https://login.hpc.ugent.be}\end{center}
+
+You do not need to install any additional software to use the web portal, a standard web browser like Firefox,
+Google Chrome or Safari (macOS) is sufficient. In addition, no SSH key pair is required to connect
+to your VSC account.
+
+Through the web portal, you can:
+\begin{itemize}
+    \item browse through your files \& directories, and inspect, manage or change them;
```

Comment:
i would add urls here (or make these "clickable")

Reply:
There's no general URL to open the file browser to your home directory, the URL includes your VSC login id.

Also, what if for some reason URLs change in OoD. That'll result in broken links in the docs.

So, I prefer not to add clickable links, the menu is pretty obvious once you're logged in to the portal.